src/convertToObject.js

```diff
@@ -5,8 +5,19 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style !== '')
```

Comment:
You don't need to compare with an empty string https://developer.mozilla.org/en-US/docs/Glossary/Truthy

Reply:
> You don't need to compare with an empty string https://developer.mozilla.org/en-US/docs/Glossary/Truthy

So, what should I do? Just remove it?

Reply:
> > You don't need to compare with an empty string https://developer.mozilla.org/en-US/docs/Glossary/Truthy
> 
> So, what should I do? I'm filtering empty string

Just check if value exists
`.filter((style) => style)`